pediatric_rootlets/generate_figure_rootlets_and_vertebral_spinal_levels.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python
+#
+# This script is used to generate a figure showing the variability between spinal levels and vertebrae levels.
+# The script also computes a distance from the PMJ to the middle of each spinal level and COV. Results are saved to
+# a CSV file.
+#
+# Usage:
+#     python generate_figure_rootlets_vertebrae_spinal_levels_draft.py -i /path/to/data_processed
+#
+# Authors: Katerina Krejci, Jan Valosek
+#
+
+import os
+import sys
+import glob
+import argparse
+
+import pandas as pd
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+import matplotlib.patheffects as pe
+
+from scipy.stats import mannwhitneyu
```

Comment:
Are you computing the Mann-Whitney test? If not, you can remove this import

Reply:
No, resolved in [c5c27b0](https://github.com/ivadomed/model-spinal-rootlets/pull/53/commits/c5c27b00464a0049cd9d69a2e7afa2a2ef5393d7).